src/ripple/net/impl/RPCCall.cpp

```diff
@@ -1221,92 +1223,91 @@ class RPCParser
 
         struct Command
         {
-            const char* name;
             parseFuncPtr parse;
             int minParams;
             int maxParams;
         };
 
-        // FIXME: replace this with a function-static std::map and the lookup
-        // code with std::map::find when the problem with magic statics on
+
         // Visual Studio is fixed.
-        static Command const commands[] = {
+        static const std::map<std::string_view, Command> command = 
```

Comment:
This data structure used to be named `commands` (with a trailing `s`).  Did you intend to change the name to singular?  Is there a good reason to name the data structure something other than `commands`?